src/molecules/Footer/index.js

```diff
@@ -0,0 +1,12 @@
+import React from "react"
+import "./footer.module.css"
+import { Link } from "gatsby"
+
+const Footer = () => (
+  <footer>
+    <Link to="/">Privacidade e Uso de Dados</Link>
```

Comment:
Change to `<Link to="/privacidade">Privacidade e uso de dados</Link>`, even that page still does not exist.

Reply:
Done